search-service/src/main/kotlin/com/egm/stellio/search/csr/model/Operation.kt

```diff
@@ -225,10 +242,25 @@ enum class Operation(val key: String, private val matchingOperationGroups: Set<O
         setOf(FEDERATION_OPS)
     );
 
-    fun getMatchingOperations(): Set<Operation> = matchingOperationGroups + this
+    fun getMatchingOperations() = matchingOperationGroups + this
+    fun getPath(entityId: URI? = null, attrId: String = "") = path?.pattern
```

Comment:
- also add the return type here
- don't get why the return type is nullable, an operation can't be without a path